src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -54,24 +55,40 @@ class CassandraJournal extends AsyncWriteJournal with CassandraRecovery with Cas
   }
 
   private def statementGroup(atomicWrite: AtomicWrite): Try[Seq[BoundStatement]] = Try {
-    atomicWrite.payload.flatMap { m =>
-      val pnr = partitionNr(m.sequenceNr)
-      val write = preparedWriteMessage.bind(m.persistenceId, pnr: JLong, m.sequenceNr: JLong, persistentToByteBuffer(m))
-      if (partitionNew(m.sequenceNr)) Seq(preparedWriteHeader.bind(m.persistenceId, pnr: JLong), write) else Seq(write)
+    // hoping to remove this in 2.4-M3 https://github.com/akka/akka/issues/18076
```

Comment:
Looks like we def can remove these in 2.4-M3 https://github.com/akka/akka/pull/18085/files#diff-0